projects/canopy/src/lib/filter-buttons/filter-button.stories.ts

```diff
@@ -0,0 +1,62 @@
+import { ReactiveFormsModule } from '@angular/forms';
+
+import { withKnobs, boolean, text } from '@storybook/addon-knobs';
+import { moduleMetadata } from '@storybook/angular';
+import { action } from '@storybook/addon-actions';
+
+import { LgFilterButtonModule } from './filter-button.module';
+import { ReactiveFormMultipleComponent } from './filter-button/filter-button-select-multiple.stories';
+import { ReactiveFormFilterSelectOneComponent } from './filter-button/filter-button-select-one.stories';
+import { notes } from './filter-button.notes';
+
+export default {
+  title: 'Components/Filter Buttons',
+  parameters: {
+    decorators: [
+      withKnobs,
+      moduleMetadata({
+        imports: [ReactiveFormsModule, LgFilterButtonModule],
+        declarations: [
+          ReactiveFormFilterSelectOneComponent,
+          ReactiveFormMultipleComponent,
+        ],
+      }),
+    ],
+    'in-dsm': {
+      id: '5f75aab1766566c38799e764',
```

Comment:
I think we can map these stories individually to the DSM components. I created the IDs as I'm not sure if you would have the privileges.

single
id: 5f75d9e2a5c09c75c88bd402
https://legalandgeneral.invisionapp.com/dsm/legalandgeneral/canopy/nav/build/asset/componentContainers/5f75d9e2a5c09cad678bd401/tab/angular?mode=edit

multi
id: 5f75d9f774621bd9596c8520
https://legalandgeneral.invisionapp.com/dsm/legalandgeneral/canopy/nav/build/asset/componentContainers/5f75d9f774621b49ea6c851f?mode=edit

I recently did it on the [icon button PR](https://github.com/Legal-and-General/canopy/pull/19/files#diff-9fb0a83f451b7408baea42e9a6b55011R163)

I think this should do it....

```js
export const selectOne = () => ({
 ...
});
selectOne.story = {
  parameters: {
    'in-dsm': {
      id: '5f75d9f774621bd9596c8520'
    }
  }
}
```
I apologise as I realise this setup is bit tricky to test!

Reply:
I am not sure how to test those, any pointers will be appreciated.:)

Reply:
to be honest it's quite tricky, the `npm run dsm-storybook:preview` might help. In terms of docs there is a bit in the repo.

https://github.com/Legal-and-General/canopy/blob/master/CONTRIBUTING.md#invision-dsm

Which links out to more comprehensive docs on invision..

https://support.invisionapp.com/hc/en-us/articles/360028510211-Configuring-the-Storybook-DSM-Integration